src/util/adb.js

```diff
@@ -271,6 +272,17 @@ export default class ADBUtils {
       value: `-profile ${deviceProfileDir}`,
     }];
 
+    for (const browser of packageIdentifiers) {
+      if (apk.startsWith(browser) && apkComponent) {
+        if (apk !== browser && !apkComponent.includes('.')) {
+          apkComponent = `${browser}.${apkComponent}`;
+        }
+        // if 'geckoview_example' has been found, leave the
+        // loop, before it checks 'geckoview'
+        break;
+      }
+    }
+
```

Comment:
Hopefully this isn't too hacky - it's mainly due to wanting to leave the loop if `geckoview_example` has been checked, so it doesn't check again on `geckoview`

Reply:
`break` after finding the matching `apk` is not a hack, it looks normal.